src/Symfony/Component/Routing/Tests/Loader/YamlFileLoaderTest.php

```diff
@@ -458,4 +460,23 @@ public function testImportingAliases()
 
         $this->assertEquals($expectedRoutes('yaml'), $routes);
     }
+
+    public function testPriorityWithPrefix()
+    {
+        new LoaderResolver([
+            $loader = new YamlFileLoader(new FileLocator(\dirname(__DIR__).'/Fixtures/localized')),
+            new class() extends AttributeClassLoader {
```

Comment:
This class doesn't exist in 5.4.
Maybe we can define the routes using plain yaml of xml instead so that we don't have to deal with this concern (migrating from annotations to attributes when merging up ;) )

Reply:
Ok, thats a good idea. I will fix

Reply:
According to docs priority can be set only in annotations or attributes. So I don't see a clear way how to test this easily on 5.4 and 6 without changing the code

```In YAML and XML you can move the route definitions up or down in the configuration file to control their priority. In routes defined as PHP attributes this is much harder to do, so you can set the optional priority parameter in those routes to control their priority```